releasing/plugin-commands-publishing/src/publish.ts

```diff
@@ -279,3 +314,20 @@ export async function runScriptsIfPresent (
     await runLifecycleHook(scriptName, manifest, opts) // eslint-disable-line no-await-in-loop
   }
 }
+
+function loadToken (helperPath: string, settingName: string) {
```

Comment:
Can you reuse the same function from `@pnpm/network.auth-header`?

Reply:
Yes, I think I need to export it